website/playground/static/playground.js

```diff
@@ -183,11 +183,7 @@ window.setupEditor = function(elements, options = {}, basePath = '', hiddenCode
     var marks = [];
     var scrollTo = false;
     class Node extends React.Component {
-      constructor(props) {
-        super(props);
-        this.toggle = this.toggle.bind(this);
-      }
-      toggle() {
+      toggle = () => {
```

Comment:
This is a not-yet standardized feature. I didn't even know it was in Chrome already. Since we're not using a transform pipeline, are you sure it's actually working?